### ISSUE TITLE
platform.linux_distribution() not available in Python 3.8

### ISSUE THREAD

**Issue by MenosGrandes** [pupil-labs/pupil#1861]
In newest python 3.8 there is no `platform.linux_distribution()` ( python issue tracker https://bugs.python.org/issue28167) and in `shared_modules/audio/__init__.py` this method is used.

Take care of it if someone know how to avoid it.

part of logs:
```
python3.8 --version
Python 3.8.2
```

```
sudo python3.8 main.py capture
world - [INFO] numexpr.utils: NumExpr defaulting to 4 threads.
world - [INFO] launchables.world: Application Version: 1.10.1772
world - [INFO] launchables.world: System Info: User: root, Platform: Linux, Machine: menosgrandes-ThinkPad-X220, Release: 5.4.0-26-generic, Version: #30-Ubuntu SMP Mon Apr 20 16:58:30 UTC 2020
world - [ERROR] launchables.world: Process Capture crashed with trace:
Traceback (most recent call last):
  File "/home/menosgrandes/Desktop/repos/pupillabupgrades/pupil_src/launchables/world.py", line 151, in world
    import audio
  File "/home/menosgrandes/Desktop/repos/pupillabupgrades/pupil_src/shared_modules/audio/__init__.py", line 68, in <module>
    if platform.linux_distribution()[0] in ("Ubuntu", "debian"):
AttributeError: module 'platform' has no attribute 'linux_distribution'
```

**Comment by papr** [comment]
@MenosGrandes Thank you very much for letting us know! Given that we want to make the code base more maintainable, we will reevaluate if the `audio` module is worth keeping. We might end up removing it completely. If not, we will add a fix as soon as we have reached a decision.

**Comment by papr** [comment]
Fixed in #1868
